idrstream/download.py

```diff
@@ -118,11 +118,10 @@ def download_image(
         """
         save_dir.mkdir(parents=True, exist_ok=True)
         image_path = self.get_IDR_mitocheck_movie_path(plate, well_num)
-        idr_location = f"{self.idr_id}@fasp.ebi.ac.uk:{image_path} "
+        idr_location = f"fasp-public@fasp.ebi.ac.uk:/pub/databases/IDR/{self.idr_index_name }/{image_path} "
 
         # -l is flag for maximum download speed (500m is 500 mb/s)
         command = f"sudo {self.aspera_path} -TQ -l500m -P 33001 -i {self.aspera_key_path} {idr_location} {save_dir}"
```

Comment:
```suggestion
        command = f"sudo {self.aspera_path} -TQ -l 500m -P 33001 -i {self.aspera_key_path} {idr_location} {save_dir}"
```
I assume there should be a space between the value and the flag, so I added a space there as a recommendation.